core/src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala

```diff
@@ -289,6 +292,42 @@ object VacuumCommand extends VacuumCommandImpl with Serializable {
 
 trait VacuumCommandImpl extends DeltaCommand {
 
+  private val supportedFsForLogging = Seq(
+    "wasbs", "wasbss", "abfs", "abfss", "adl", "gs", "file"
```

Comment:
I have a question: what is the relationship between whether log vacuum and the file system used?

Reply:
+1 to make this a config. The list is also missing hdfs.

Before this change, vacuum doesn't make a commit, so people can run vacuum concurrently with any other queries. After this change, this could be dangerous, for example on S3, if people still run vacuum concurrently with any other queries without using a right LogStore. Hence, we don't enable it by default.